server/chart_config.json

```diff
@@ -1142,6 +1210,38 @@
       "scale": false
     }
   },
+  {
+    "category": "Demographics",
+    "titleId": "Literacy",
+    "title": "Literacy",
+    "description": "Number of literate and illiterate population",
```

Comment:
Just "Literate and Illiterate Population"  (i.e., No "Number of")

Reply:
Done